crates/languages/src/rust/highlights.scm

```diff
@@ -116,3 +121,52 @@
   (line_comment)
   (block_comment)
 ] @comment
+
+[
+  "!"
+  "!="
+  "%"
+  "%="
+  "&"
+  "&="
+  "&&"
+  "*"
+  "*="
+  "*"
+  "+"
+  "+="
+  ","
+  "-"
+  "-="
+  "->"
+  "."
+  ".."
+  "..="
+  "..."
+  "/"
+  "/="
+  ":"
+  ";"
+  "<<"
+  "<<="
+  "<"
+  "<="
+  "="
+  "=="
+  "=>"
+  ">"
+  ">="
+  ">>"
+  ">>="
+  "@"
+  "^"
+  "^="
+  "|"
+  "|="
+  "||"
+  "?"
+  "::"
+  "#"
```

Comment:
I don't think `#` makes sense as an operator, as it doesn't seem like you'd want to style it the same as other operators.

Given that it's not in Rust's list of operators, seems like it shouldn't be included here.

Reply:
Thanks, I think both of those sound reasonable to me!